guides/serialization_and_saving.py

```diff
@@ -21,10 +21,10 @@
 The Keras API makes it possible to save all of these pieces to disk at once,
 or to only selectively save some of them:
 
-- Saving everything into a single archive in the TensorFlow SavedModel format
+- Saves everything into a single archive in the TensorFlow SavedModel format
```

Comment:
Please keep "saving" in this list, it's what's intended.

Reply:
I have Updated it with "Saving". Thank you!